code/modules/mob/living/simple_animal/abnormality/he/der_freischutz.dm

```diff
@@ -40,7 +40,7 @@
 	var/targetx
 	var/targety
 	for(var/mob/M in GLOB.mob_living_list)
-		if(istype(M,/mob/living/simple_animal/bot) || istype(M,/mob/living/simple_animal/hostile/abnormality/der_freischutz) || (src.z != M.z) || (M.stat == DEAD))
+		if(istype(M,/mob/living/simple_animal/bot) || istype(M,/mob/living/simple_animal/hostile/abnormality/der_freischutz) || (src.z != M.z) || (M.stat == DEAD) || (M.status_flags & GODMODE))
```

Comment:
Since you are already checking for godmode.
```suggestion
		if(istype(M,/mob/living/simple_animal/bot) || (src.z != M.z) || (M.stat == DEAD) || (M.status_flags & GODMODE))
```